tests/server/test_startup.py

```diff
@@ -93,6 +93,21 @@ def _get_raw_params(path):
         }
 
 
+@pytest.mark.asyncio
+async def test_cluster_connect_error(base_url, http_client):
+    url = "{}/api/config/connection/".format(base_url)
+    conn_details = {
+        'connection': {
+            'type': 'TCP',
+            'address': 'tcp://unknown:8786',
+        }
+    }
```

Comment:
:+1: - instead of "unknown", you could also [use the "invalid" name that is guaranteed to not resolve to any hostname](https://tools.ietf.org/html/rfc6761#section-6.4) (RFC 6761). It should also guarantee that the name resolution fails early, so the error should happen earlier ("Name resolution APIs and libraries SHOULD recognize "invalid" names as special and SHOULD always return immediate negative responses.  Name resolution APIs SHOULD NOT send queries for "invalid" names to their configured caching DNS server(s).")

Maybe this will also fix [the timeout issue here](https://travis-ci.org/github/LiberTEM/LiberTEM/jobs/670861793?utm_medium=notification&utm_source=github_status)?

Reply:
"invalid" address is raising a `ValueError: missing port number in address 'invalid'`. It only takes around 8 ms to respond. For "invalid:8080" it takes 10s to respond.

Reply:
Very nice - yeah I think we can't get around the 10s timeout without doing something funky (mocking the `Executor` comes to mind...) - I guess that's a retry-loop somewhere in Dask that retries even the DNS lookup :grinning: 

I like the new solution, let's keep it :+1: